packages/dd-trace/src/ci-visibility/dynamic-instrumentation/worker/index.js

```diff
@@ -0,0 +1,106 @@
+const { parentPort } = require('worker_threads')
+// TODO: move session to common place
+const session = require('../../../debugger/devtools_client/session')
+// TODO: move getLocalStateForCallFrame to common place
+const { getLocalStateForCallFrame } = require('../../../debugger/devtools_client/snapshot')
```

Comment:
this might be a bit trickier but no reason not to do it

Reply:
Especially if we use two different `session.js` files as I just laid out in my previous comment. If so, we need a way to pass in the current `session` object. So for now, I'd just leave this as-is